src/app/components/simple-editable-list/SimpleEditableList.jsx

```diff
@@ -50,11 +50,11 @@ export default class SimpleEditableList extends Component {
     render() {
         return (
             <div>
-                {this.props.fields.length ? (
+                {this.props.fields?.length ? (
```

Comment:
Does that operator work between fields and length?

Reply:
It's a `null` check

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining